third_party/accessibility/ax/ax_node.h

```diff
@@ -436,6 +440,8 @@ class AX_EXPORT AXNode final {
   // Finds and returns a pointer to ordered set containing node.
   AXNode* GetOrderedSet() const;
 
+  AXNode* GetLowestPlatformAncestor() const;
```

Comment:
Can we keep Chromium's comment?

```suggestion
  // If this node is exposed to the platform's accessibility layer, returns this
  // node. Otherwise, returns the lowest ancestor that is exposed to the
  // platform. (See `IsLeaf` and `IsIgnored` for information on what is
  // exposed to platform APIs.)
  AXNode* GetLowestPlatformAncestor() const;
```